index.html

```diff
@@ -275,8 +275,25 @@ <h3 class="im-qaisar">Glad to see you!</h3>
             <li> <img src="./public/adobe-illustrator.svg" alt=""> <span>Adobe Illustrator</span> </li>
           </ul>
         </div>
+
     </section>
+    <section class="form">
+      <h1>Contact Me</h1>
+      <p>If you have an application, you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+
+      <form>
+          <label for="name"></label>
+          <input class="input" type="text" id="name" name="name" placeholder="Qaisar Abbas" required><br><br>
+
+          <label for="email"></label>
+          <input class="input" type="email" id="email" name="email" placeholder="kayser.abbas@gmail.com" required><br><br>
 
+          <label for="message"></label>
+          <textarea class="input" id="message" name="message" rows="6" cols="35" placeholder="Write message here" required></textarea><br><br>
+
```

Comment:
one of the requirements is maximum length of char.
You can add min and max char requirements by minlength="1" maxlength="500"

Reply:
Thanks Noted.